core/templates/dev/head/pages/exploration-editor-page/exploration-editor-page.controller.ts

```diff
@@ -20,6 +20,7 @@
 // TODO(vojtechjelinek): this block of requires should be removed after we
 // introduce webpack for /extensions
 require('components/ck-editor-helpers/ck-editor-4-rte.directive.ts');
+require('components/ck-editor-helpers/ck-editor-5-rte.directive.ts');
```

Comment:
If we can rename the file as `ck-editor-5-rte`, why can we rename variable as 'CkEditor5Rte`? (is there any strict rule to have file names only with small caps?)

[Asking this just out of curiosity.]

Reply:
I see comment for the same over [here](https://github.com/oppia/oppia/pull/7184/files#r307957158). Marking resolved.